cmd/report_test.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.k6.io/k6/execution"
+	"go.k6.io/k6/lib"
+	"go.k6.io/k6/lib/executor"
+	"go.k6.io/k6/lib/testutils"
+	"go.k6.io/k6/lib/types"
+	"gopkg.in/guregu/null.v3"
+)
+
+func TestCreateReport(t *testing.T) {
+	t.Parallel()
+	importedModules := []string{
+		"k6/http",
+		"my-custom-module",
+		"k6/experimental/webcrypto",
+		"file:custom-from-file-system",
+		"k6",
+		"k6/x/custom-extension",
+	}
+
+	outputs := []string{
+		"json",
+		"xk6-output-influxdb",
+	}
+
+	logger := testutils.NewLogger(t)
+	opts, err := executor.DeriveScenariosFromShortcuts(lib.Options{
+		VUs:      null.IntFrom(10),
+		Duration: types.NullDurationFrom(5 * time.Minute),
```

Comment:
Nit; If I'm not wrong, this is only needed to have `constant-vus` as executor. However, imho this is a bit confusing, because it's used nowhere, and differs from the actual execution (~10ms) and the actual amount of VUs (6).

So, I'd suggest to either make it match with the values below (perhaps extracting these values as a shared variable?), or just removing these lines and switching the executor type in the assertion, which is even simpler/shorter, thus easier to understand.